source/sass/component/_segment.scss

```
@@ -58,7 +58,7 @@ $c-segment-color-overlay: var(--c-segment-color-overlay, $color-alpha) !default;
 .c-segment.c-segment--text {
     &-large .c-segment__content {
         @include mq(map_get($breakpoints-map, "md")) {
-            font-size: 1.2em; 
+            font-size: 1.2rem; 
```

This needs to be relative. 
Option allows for large text in segments. 

```suggestion
            font-size: 1.2em;
```